public/components/trace_analytics/components/common/search_bar.tsx

```diff
@@ -36,22 +36,21 @@ export const renderDatePicker = (
   );
 };
 
-export interface SearchBarProps extends FiltersProps {
+export interface SearchBarProps {
+  filters: FilterType[];
+  setFilters: (filters: FilterType[]) => void;
   query: string;
   setQuery: (query: string) => void;
   startTime: string;
   setStartTime: (startTime: string) => void;
   endTime: string;
   setEndTime: (endTime: string) => void;
-}
-
-interface SearchBarOwnProps extends SearchBarProps {
```

Comment:
i think some component prop definitions depend on `SearchBarProps` to not include functions like `refresh`

Reply:
My understanding was this was necesary as i needed to sepearate the Ui elements to prevent it from creating an additional filter.
<img width="1083" alt="Screenshot 2024-08-19 at 2 07 33 PM" src="https://github.com/user-attachments/assets/13121260-1a8c-411d-b132-490eec4a6663">
By separating:
<img width="1083" alt="Screenshot 2024-08-19 at 2 08 02 PM" src="https://github.com/user-attachments/assets/896e0427-4bae-482c-9170-e5d5dd6344e9">